File: importado-do-antigo/ALGORITMOS-E-ESTRUTURAS-DE-DADOS-II (PT2)/#Teorico/Códigos de Estudos/3 - Estruturas de Dados Lineares e Flexíveis/src/Inteiro.java
public class Inteiro {
    private int valor;

    public Inteiro() {
        valor = 0;
    }

    public Inteiro(int i) {
        valor = i;
    }

    public int getValor() {
        return valor;
    }

    public void setValor(int valor) {
        this.valor = valor;
    }

    public void imprimir() {
        System.out.println("Valor: " + this.valor);
    }

    public String toString() {
        return "Valor: " + this.valor;
    }
}

File: importado-do-antigo/ALGORITMOS-E-ESTRUTURAS-DE-DADOS-II (PT3)/Algoritmos e Estruturas de Dados II - TEO/TAD - Tipo Abstrato de Dados/ListaEncadeada/ListaEncadeada/Inteiro.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ListaEncadeada{
	class Inteiro{

		private int valor;
		public int Valor{
			get { return valor; }
			set { valor = value; }
		}

		public Inteiro(int valor){
			this.valor = valor;
		}

		public Inteiro(){
			valor = 0;
		}

		public void imprimir(){
			Console.WriteLine("Valor -> " + valor);
		}
	}
}
